dashboard/app/helpers/levels_helper.rb

```diff
@@ -233,6 +233,7 @@ def blockly_options(local_assigns={})
       project_template_level_name
       scrollbars
       original_start_blocks
+      is_project
```

Comment:
maybe call this is_project_level or has_project to reduce confusion over what a project "is"?